### ISSUE TITLE
Hide keys

### ISSUE THREAD

**Issue by TuriGuilano** [Denniswegereef/web-app-from-scratch-18-19#5]
Hide keys with .env

https://github.com/Denniswegereef/web-app-from-scratch-18-19/blob/e5791c01316ef9446663c8f592654415e34e9f0c/src/js/helper/api-module.js#L5

**Comment by Denniswegereef** [comment]
For now I'm leaving this open till I find a elegant solution for it. any tips?

**Comment by rijkvanzanten** [comment]
Yes, do not keep it in your repo for any second longer than needed. 

You should:
* Remove it from the file
* Invalidate the LastFM key in your LastFM account
* Generate a new key

ASAP

After that, you can add a new file `config.js` that you import from the `api.js` file. You can add this `config.js` file to the `.gitignore` file to keep it out of your repo.
